File: src/main/java/com/deneb/client/features/modules/player/AntiForceLook.java
package com.deneb.client.features.modules.player;

import com.deneb.client.event.events.client.PacketEvent;
import com.deneb.client.features.Category;
import com.deneb.client.features.Module;
import net.minecraft.network.play.server.SPacketPlayerPosLook;
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;

/**
 * Created by 086 on 12/12/2017.
 */
@Module.Info(name = "AntiForceLook", category = Category.PLAYER)
public class AntiForceLook extends Module {

    @SubscribeEvent
    public void onPacket(PacketEvent.Receive event){
        if (event.getPacket() instanceof SPacketPlayerPosLook) {
            SPacketPlayerPosLook packet = (SPacketPlayerPosLook) event.getPacket();
            packet.yaw = mc.player.rotationYaw;
            packet.pitch = mc.player.rotationPitch;
        }
    }

}

File: src/main/java/club/deneb/client/features/modules/player/AntiForceLook.kt
package club.deneb.client.features.modules.player

import club.deneb.client.event.events.client.PacketEvent
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
import net.minecraft.network.play.server.SPacketPlayerPosLook
import club.deneb.client.features.AbstractModule
import club.deneb.client.features.Category
import club.deneb.client.features.Module

@Module.Info(name = "AntiForceLook", category = Category.PLAYER)
class AntiForceLook : Module() {
    @SubscribeEvent
    fun onPacket(event: PacketEvent.Receive) {
        if (event.getPacket() is SPacketPlayerPosLook) {
            val packet = event.getPacket() as SPacketPlayerPosLook
            packet.yaw = mc.player.rotationYaw
            packet.pitch = mc.player.rotationPitch
        }
    }
}